sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2434,10 +2438,12 @@ def __init__(
       gcs_location=None,
       method=None,
       use_native_datetime=False,
+      output_type=None,
       *args,
       **kwargs):
     self.method = method or ReadFromBigQuery.Method.EXPORT
     self.use_native_datetime = use_native_datetime
+    self.output_type = output_type
```

Comment:
It looks like this isn't actually used anywhere. We should inspect this variable when the transform is expanded, and if it is `"BEAM_ROW"`, we should use your new utilities to add a ParDo at the end that converts the dictionaries to the generated user type. Does that make sense?